bibedit/volatile_check_accelerator.xml_ignore

```diff
@@ -0,0 +1,13 @@
+<!-- BibEdit-Template-Name: accelerator -->
+<!-- BibEdit-Template-Description: accelerator -->
+<record>
+  <datafield tag="046" ind1=" " ind2=" ">
+    <subfield code="x">VOLATILE:yes</subfield>
+  </datafield>
+  <datafield tag="702" ind1=" " ind2=" ">
+    <subfield code="z">VOLATILE:current</subfield>
+  </datafield>
+  <datafield tag="856" ind1="4" ind2=" ">
+    <subfield code="y">VOLATILE:home page</subfield>
+  </datafield>
+</record>
```

Comment:
that's only the ignore file, not the full template, which is [here](https://github.com/inspirehep/inspire/pull/307/files#diff-1a2fba5045ff1df12af5ee4a2b1a7c81)